### PR TITLE
Change codeblock to recommended style (1)

### PR DESCRIPTION
**Pull request by sypets**
Resolves: #278

### CODE REVIEW

File: Documentation/Appendix/Linux/SettingUpTypo3ManuallyLinux.rst [hunk 1]
```diff
@@ -33,7 +33,7 @@ Setup
 Apache Webserver
 -----------------
 
-::
+.. code-block:: bash
```
Comment (linawolf) [inline Documentation/Appendix/Linux/SettingUpTypo3ManuallyLinux.rst:36]: shall we add captions here?

Comment (sypets) [reply Documentation/Appendix/Linux/SettingUpTypo3ManuallyLinux.rst:36]: done.

**Review by sypets — COMMENTED** [review]
Also removed the highlight directive and checked for code-block:: none in this PR.

This one should be ok.

File: Documentation/Appendix/Linux/SettingUpTypo3ManuallyLinux.rst [hunk 2]
```diff
@@ -53,29 +53,37 @@ PHP 7.4
 .. important::
    Make sure you use the correct PHP version for your TYPO3 version.
 
-Make some changes to php.ini and reload::
+Make some changes to php.ini (or appropriate file in :file:`conf.d`) and reload:
+
+.. code-block:: bash
```
Comment (mbrodala) [inline Documentation/Appendix/Linux/SettingUpTypo3ManuallyLinux.rst:58]:
```suggestion
.. code-block:: ini
```

**Review by mbrodala — CHANGES_REQUESTED** [review]

**Review by linawolf — APPROVED** [review]

**Review by mbrodala — APPROVED** [review]

**Pull request merged**